src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestHelpers/TestRunner.cs

```diff
@@ -44,25 +44,25 @@ public static string GetDefaultTestDir()
 
         public string TestRoot { get; set; } = GetDefaultTestDir();
 
-        public void AddDir(bool numberIsFloat = false, string directory = "")
+        public void AddDir(Dictionary<string, bool> setup, string directory = "")
```

Comment:
>Dictionary<string, bool> setup [](http://example.com/codeflow?start=27&length=30)

This may require updates to other runners (such as SQL and Power Apps)